Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:3.8 
+
+WORKDIR /usr/src/app 
+
+COPY .synapseConfig ./
```

Comment:
Don't do this.  Use an environmental variable: 

```
SYNAPSE_AUTH_TOKEN='<my_personal_access_token>' 
```

Reply:
`docker run -e SYNAPSE_AUTH_TOKEN='PAT here' image`

Reply:
- removed copy command fo .synapseConfig in DockerFile
- modified synapse login to default in __main__.py which will search for env variable auth token
- modified docker run command and installation instructions in readme to require specification of env variable with auth token